chips/lowrisc/src/registers/adc_ctrl_regs.rs

```diff
@@ -0,0 +1,129 @@
+// Generated register constants for adc_ctrl
+
+// Built for earlgrey_silver_release_v5-5548-g933e261c9
+// https://github.com/lowRISC/opentitan/tree/933e261c91e67309b242d87a97e42b8af1bd17ef
+// Tree status: modified
```

Comment:
Can we generate these from a clean tree as well?

Reply:
Oops, how embarassing.  Fixed.